funnel/templates/proposal_comment_to_proposer_email.md.jinja2

```diff
@@ -1,6 +0,0 @@
-**{{ current_auth.user.pickername }}** replied to you on **{{ proposal.title
-}}**.
-
-{{ comment.message }}
-
-View it on [funnel]({{link}})
```

Comment:
We are not sending email when user comments on a proposal?

Reply:
It was removed last month due to spam.